docs/2_2_common_schema.adoc

```diff
@@ -934,7 +975,7 @@ _Discussion of the combinations that are [underline]#not allowed#_:
 |_The combinations `"constant" / "parameter"`, `"constant" / "calculatedParameter"` and `"constant" / "input"` do not make sense, since parameters and inputs are set from the environment, whereas a constant has always a value._
 
 ^|_[red]#(b)#_
-|_The combinations `"discrete / parameter"`, `"discrete / calculatedParameter"`, `"continuous / parameter"` and `"continuous" / "calculatedParameter"` do not make sense, since `causality = "parameter"` and `"calculatedParameter"` define variables that do not depend on time, whereas `"discrete"` and `"continuous"` define variables where the values can change during simulation._
+|_The combinations `"discrete / structuralParameter"`, `"discrete / parameter"`, `"discrete / calculatedParameter"`, `"continuous / structuralParameter", `"continuous / parameter"` and `"continuous" / "calculatedParameter"` do not make sense, since `causality = "structuralParameter"`, causality = "parameter"` and `"calculatedParameter"` define variables that do not depend on time, whereas `"discrete"` and `"continuous"` define variables where the values can change during simulation._
```

Comment:
This should be reformulated in plain english (is or equals vs. =)

Reply:
I did this part before our decision to use plain english.
However, switching to plain english seems to be a bigger topic. (`=` for attribute values is used at several places). I think we should not mix up both changes. This pu request should only cover the content that was missing to cover arrays. There should be a general issue to harmonize equal vs `=` in the specification.